StarMapService/EdsmSystemData.cs

```diff
@@ -69,6 +69,38 @@ public static List<StarSystem> GetStarMapSystems(string[] systems, bool showCoor
             return null;
         }
 
+        /// <summary> Partial of system name is required. </summary>
+        public static List<StarSystem> GetStarMapSystemsPartial(string system, bool showCoordinates = true, bool showSystemInformation = true)
+        {
+            if (system == null) { return null; }
+            var client = new RestClient(baseUrl);
+            var request = new RestRequest("api-v1/systems", Method.POST);
+            request.AddParameter("systemName", system + "%");
```

Comment:
Is "%" necessary? The API doesn't document that as a requirement for the `https://www.edsm.net/api-v1/systems` endpoint (ref. https://www.edsm.net/en/api-v1)

Reply:
It looks like both work, but it is unnecessary. Compare:
https://www.edsm.net/api-v1/systems?systemName=Ach
https://www.edsm.net/api-v1/systems?systemName=Ach%

Reply:
It's absolutely necessary, per Anthor.

Reply:
After further discussion with @Hoodathunk, it appears to be necessary to add to all 2 letter searches (at minimum). This is an undocumented API requirement that we should note in the code.